sentry-android-core/src/main/java/io/sentry/android/core/ActivityLifecycleIntegration.java

```diff
@@ -531,6 +540,17 @@ private void onFullFrameDrawn() {
     cancelTtfdAutoClose();
   }
 
+  private void finishExceededTtfdSpan(final @Nullable ISpan ttidSpan) {
+    if (ttfdSpan == null) {
+      return;
+    }
+    ttfdSpan.setDescription(getExceededTtfdDesc(ttfdSpan));
```

Comment:
`low`: I guess it's never happening but in case `finishExceededTtfdSpan` get's called mutliple times, we'd add the `"-exceeded"` multiple times here.